### ISSUE TITLE
abut train corpus format

### ISSUE THREAD

**Issue by Rooders** [THUNLP-MT/Document-Transformer#10]
hello~ 
When I use this code to training a model, What format should be processed for the source corpus, the target corpus, the context corpus? are they  tokenized and BPE? Could u send me a demo about  it? 
Thank u very mach.

**Comment by Glaceon31** [comment]
Using the same processing steps as standard NMT systems is OK (e.g. tokenization and BPE for English)

You may refer to the user manual at https://github.com/THUNLP-MT/THUMT for detail

**Comment by Rooders** [comment]
> Using the same processing steps as standard NMT systems is OK (e.g. tokenization and BPE for English)
> 
> You may refer to the user manual at https://github.com/THUNLP-MT/THUMT for detail

thank u very much 
I meet anther issue now, when I use THUMT code to train a standard sentence-level Transformer model,it achieved only 29 BLEU score in validation set and my training  set is  LDC zh-en corpus(2M sentence-pair), validation set is MT06, which is same as your paper,but haven‘t got same BLEU score(it is 48.09 in your paper).
So I guess my  parameters setting maybe not same as yours ,under is mine,if it is impossible ,can u sent your parameter set?
![image](https://user-images.githubusercontent.com/38709093/91650431-11af4980-eab2-11ea-86fb-e55eb3de19a7.png)

**Comment by Rooders** [comment]
> > Using the same processing steps as standard NMT systems is OK (e.g. tokenization and BPE for English)
> > You may refer to the user manual at https://github.com/THUNLP-MT/THUMT for detail
> 
> thank u very much
> I meet anther issue now, when I use THUMT code to train a standard sentence-level Transformer model,it achieved only 29 BLEU score in validation set and my training set is LDC zh-en corpus(2M sentence-pair), validation set is MT06, which is same as your paper,but haven‘t got same BLEU score(it is 48.09 in your paper).
> So I guess my parameters setting maybe not same as yours ,under is mine,if it is impossible ,can u sent your parameter set?
> ![image](https://user-images.githubusercontent.com/38709093/91650431-11af4980-eab2-11ea-86fb-e55eb3de19a7.png)

My e-mali adr :

> > Using the same processing steps as standard NMT systems is OK (e.g. tokenization and BPE for English)
> > You may refer to the user manual at https://github.com/THUNLP-MT/THUMT for detail
> 
> thank u very much
> I meet anther issue now, when I use THUMT code to train a standard sentence-level Transformer model,it achieved only 29 BLEU score in validation set and my training set is LDC zh-en corpus(2M sentence-pair), validation set is MT06, which is same as your paper,but haven‘t got same BLEU score(it is 48.09 in your paper).
> So I guess my parameters setting maybe not same as yours ,under is mine,if it is impossible ,can u sent your parameter set?
> ![image](https://user-images.githubusercontent.com/38709093/91650431-11af4980-eab2-11ea-86fb-e55eb3de19a7.png)

My email : hangsonghk@gmail.com

**Comment by Rooders** [comment]
> > Using the same processing steps as standard NMT systems is OK (e.g. tokenization and BPE for English)
> > You may refer to the user manual at https://github.com/THUNLP-MT/THUMT for detail
> 
> thank u very much
> I meet anther issue now, when I use THUMT code to train a standard sentence-level Transformer model,it achieved only 29 BLEU score in validation set and my training set is LDC zh-en corpus(2M sentence-pair), validation set is MT06, which is same as your paper,but haven‘t got same BLEU score(it is 48.09 in your paper).
> So I guess my parameters setting maybe not same as yours ,under is mine,if it is impossible ,can u sent your parameter set?
> ![image](https://user-images.githubusercontent.com/38709093/91650431-11af4980-eab2-11ea-86fb-e55eb3de19a7.png)

My e-mali : hangsonghk@gmail.com 
Thank you very mach

**Comment by Rooders** [comment]
> > Using the same processing steps as standard NMT systems is OK (e.g. tokenization and BPE for English)
> > You may refer to the user manual at https://github.com/THUNLP-MT/THUMT for detail
> 
> thank u very much
> I meet anther issue now, when I use THUMT code to train a standard sentence-level Transformer model,it achieved only 29 BLEU score in validation set and my training set is LDC zh-en corpus(2M sentence-pair), validation set is MT06, which is same as your paper,but haven‘t got same BLEU score(it is 48.09 in your paper).
> So I guess my parameters setting maybe not same as yours ,under is mine,if it is impossible ,can u sent your parameter set?
> ![image](https://user-images.githubusercontent.com/38709093/91650431-11af4980-eab2-11ea-86fb-e55eb3de19a7.png)

My e-mail : hangsonghk@gmail.com 
Thank you very mach

**Comment by Glaceon31** [comment]
> > Using the same processing steps as standard NMT systems is OK (e.g. tokenization and BPE for English)
> > You may refer to the user manual at https://github.com/THUNLP-MT/THUMT for detail
> 
> thank u very much
> I meet anther issue now, when I use THUMT code to train a standard sentence-level Transformer model,it achieved only 29 BLEU score in validation set and my training set is LDC zh-en corpus(2M sentence-pair), validation set is MT06, which is same as your paper,but haven‘t got same BLEU score(it is 48.09 in your paper).
> So I guess my parameters setting maybe not same as yours ,under is mine,if it is impossible ,can u sent your parameter set?
> ![image](https://user-images.githubusercontent.com/38709093/91650431-11af4980-eab2-11ea-86fb-e55eb3de19a7.png)

It seems that you only use one reference in validation when NIST test sets have 4 references. Using more references will result in higher BLEU scores.
